src/misc/fs_utils_win32.cpp

```diff
@@ -256,9 +256,49 @@ void set_dos_file_time(const NativeFileHandle handle, const uint16_t date,
 	SetFileTime(handle, nullptr, nullptr, &write_time);
 }
 
+// Includes a Windows specific hack
+// We move the file to be deleted to a temp file before delete
+// This allows a file with the same name to be created even if there are open file handles
+// See bug report about the game Abuse: https://github.com/dosbox-staging/dosbox-staging/issues/4123
```

Comment:
Please add periods at the end of each sentence. I'm not (just) being pedantic 😄; if someone reformats this later into a single paragraphs, it will be hard to see where the sentences end. This happened to me previously when reformatting stuff.

Reply:
Yeah, you mentioned that before.  Sorry I forgot.  Added periods everywhere.

Reply:
> Yeah, you mentioned that before. Sorry I forgot. Added periods everywhere.

Thanks man 🙇🏻‍♂️  😄